packages/shared-lib/src/package-manager/package.ts

```diff
@@ -172,6 +177,78 @@ export namespace ExpectedPackage {
 			}
 		}[]
 	}
+	export interface ExpectedPackageHtmlTemplate extends Base {
+		type: PackageType.HTML_TEMPLATE
+		content: {
+			/** path to the HTML template */
+			path: string
+			/** Add prefix to output artifacts */
+			outputPrefix: string
```

Comment:
What does this add a prefix _to_? the url filename of the artifact?

Reply:
Yes, the output of this package are defined by the various defined steps. This adds a prefix to them. Hmm, perhaps we should remove this altogether..?